lib/kitchen/directions/bake_all_numbered_exercise_types.rb

```diff
@@ -14,9 +14,7 @@ def self.v1(within:, exercise_options: {}, question_options: {})
               exercise: exercise, number: exercise_counter, options: exercise_options
             )
           else
-            question_options[:number] = exercise_counter
-            question_options[:question] = exercise
-            BakeInjectedExerciseQuestion.v1(question_options)
+            BakeInjectedExerciseQuestion.v1(number: exercise_counter, question: exercise)
```

Comment:
there may be more question options than `number` and `question` passed in, see [marketing](https://github.com/openstax/cookbook/blob/main/lib/recipes/marketing/bake#L62)